networkit/viz.pyx

```diff
@@ -60,21 +61,30 @@ cdef class GraphLayoutAlgorithm:
 		self._G = None # just to be sure the graph is deleted
 
 	def numEdgeCrossings(self):
-		""" Computes approximation (in parallel) of the Spanning Edge Centrality. """
+		""" Returns the number of computed edge crossings in the layout. """
 		if self._this == NULL:
 			raise RuntimeError("Error, object not properly initialized")
 		return self._this.numEdgeCrossings()
 
 	def getCoordinates(self):
-		""" Computes approximation (in parallel) of the Spanning Edge Centrality. """
+		""" Get node coordinates for computed layout."""
 		if self._this == NULL:
 			raise RuntimeError("Error, object not properly initialized")
-		cdef pair[double, double] pr = pair[double, double](0, 0)
+		cdef vector[double] pr3d = [0, 0, 0]
+		cdef vector[double] pr2d = [0, 0]
```

Comment:
Can't we create `pr3d` or `pr2d` depending on whether `num_dim` is 3 or 2 (by moving them inside the `if` statement below)?